src/fpd/enrichment.js

```diff
@@ -51,6 +52,11 @@ export const enrichFPD = hook('sync', (fpd) => {
         deepSetValue(ortb2, 'device.ext', Object.assign({}, ext, ortb2.device.ext));
       }
 
+      const documentLang = dep.getDocument().documentElement.lang;
+      if (documentLang) {
+        deepSetValue(ortb2, 'ext.prebid.bidRequest.params.documentLang', documentLang);
```

Comment:
How do we feel about site.content.ext.documentLang or site.ext.data.documentLang?